db/migrate/20180209172420_create_reviews.rb

```diff
@@ -0,0 +1,9 @@
+class CreateReviews < ActiveRecord::Migration[5.1]
+  def change
+    create_table :reviews do |t|
+      t.string :user_name
+      t.text :body
+      t.references :book, foreign_key: true
+    end
+  end
+end
```

Comment:
You'd want the reviews table to essentially be a joint able between book and user. Rather than storing a username on a review, I'd want it to be associated with a User.